js/redux/modules/reducers/surveyReducer.js

```diff
@@ -10,6 +10,8 @@ const initialState = {
 
 export function SurveyReducer(state = initialState, action) {
   switch (action.type) {
+  case 'RESET_SURVEY': 
```

Comment:
Use the default case as a fallback
Ex:

`case 'RESET_SURVEY':
default: 
  return state`

Reply:
Ah, should of thought of that. Thanks, Eddie.